datadog/tests/data_source_datadog_metric_tags_test.go

```diff
@@ -0,0 +1,36 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccDatadogMetricTagsDatasource(t *testing.T) {
+	t.Parallel()
+	_, _, accProviders := testAccFrameworkMuxProviders(context.Background(), t)
+	metric := "datadog.agent.running"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV5ProviderFactories: accProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDatasourceMetricTagsConfig(metric),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.datadog_metric_tags.agent", "metric", metric),
+				),
+			},
+		},
+	})
+}
+
+func testAccDatasourceMetricTagsConfig(metric string) string {
+	return fmt.Sprintf(`
+data "metric_tags" "agent" {
```

Comment:
```suggestion
data "datadog_metric_tags" "agent" {
```

`metric_tags` is some other provider datasource.

Once you've done this fix, can you please generate the cassettes for this test as explained in [DEVELOPMENT.md](https://github.com/DataDog/terraform-provider-datadog/blob/master/DEVELOPMENT.md#testing-the-provider)?

Reply:
Thanks @fpighi; it's a been a long week - I've pushed additional commits correcting the typo, and adding relevant cassettes.